libraries/chain/asset_evaluator.cpp

```diff
@@ -45,6 +58,8 @@ void_result asset_create_evaluator::do_evaluate( const asset_create_operation& o
    FC_ASSERT( op.common_options.whitelist_authorities.size() <= chain_parameters.maximum_asset_whitelist_authorities );
    FC_ASSERT( op.common_options.blacklist_authorities.size() <= chain_parameters.maximum_asset_whitelist_authorities );
 
+   detail::check_asset_options_hf_1268(d.head_block_time(), op.common_options);
```

Comment:
This check can be removed safely after HF and also the whole `check_asset_options_hf_1268` function after the hardfork time pass. Please add `Todo:` comments around them so we can remove at any hardfork after `HARDFORK_1268_TIME`

Reply:
Done.